feature-libs/product-configurator/rulebased/components/tab-bar/configurator-tab-bar.component.ts

```diff
@@ -96,13 +97,16 @@ export class ConfiguratorTabBarComponent {
    */
   navigateToOverview(routerData: ConfiguratorRouter.Data) {
     this.routingService
-      .go({
-        cxRoute: 'configureOverview' + routerData.owner.configuratorType,
-        params: {
-          entityKey: routerData.owner.id,
-          ownerType: routerData.owner.type,
+      .go(
+        {
+          cxRoute: 'configureOverview' + routerData.owner.configuratorType,
+          params: {
+            entityKey: routerData.owner.id,
+            ownerType: routerData.owner.type,
+          },
         },
-      })
+        { queryParams: { productCode: this.getProductCode(routerData) } }
```

Comment:
By switching between `Configurator` and `Overview` tabs productCode should be delivered